control/vehicle_cmd_gate/src/vehicle_cmd_gate.cpp

```diff
@@ -572,7 +579,26 @@ AckermannControlCommand VehicleCmdGate::filterControlCommand(const AckermannCont
 
   is_filter_activated.stamp = now();
   is_filter_activated_pub_->publish(is_filter_activated);
-  filter_activated_marker_pub_->publish(createMarkerArray(is_filter_activated));
+  filter_activated_marker_raw_pub_->publish(createMarkerArray(is_filter_activated));
```

Comment:
Could you separate the marker-related code into separate functions?

Reply:
I have refactored the code related to marker publishing by separating it into a new function called publishMarkers.
https://github.com/autowarefoundation/autoware.universe/pull/5426/commits/00bf3d996cdebafebb9e028ef330996fab3b92c0